README.md

```diff
@@ -59,8 +60,9 @@ First make sure you've got Rust **1.4.0** or greater available, then:
 
 `cargo test` to run all tests.
 
-To run rustfmt after this, use `cargo run -- filename`. See the notes above on
-running rustfmt.
+To run rustfmt after this, use `cargo run --bin rustfmt -- filename`. See the
+notes above on running rustfmt. To run cargo-fmt, use
+`cargo run --bin cargo-fmt`
```

Comment:
This seems confusing, to run, shouldn't it be `cargo fmt`? And to install, I'd have thought I need to do something `cargo install --bin cargo-fmt` or something, rather than `cargo fmt`.